src/main/python/main/ayab/engine/options.py

```
@@ -65,8 +65,8 @@ class OptionsTab(SignalSender, QWidget):
     @date   June 2020
     """
 
-    lowercase_e_rc
-    lowercase_e_reversed_rc
+    eKnitter_rc
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)